Examples/OneSignalDemo/.idea/encodings.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Remove all .idea files. Add to .gitignore too

Reply:
Got it